alarms/consumer/brokers/brokers_nats.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build !rabbitmq
+// +build !rabbitmq
+
+package brokers
+
+import (
+	"context"
+	"log/slog"
+
+	"github.com/absmach/supermq/pkg/messaging"
+	broker "github.com/absmach/supermq/pkg/messaging/nats"
+)
+
+const AllTopic = "alarms.>"
+
+func NewPubSub(ctx context.Context, url string, logger *slog.Logger) (messaging.PubSub, error) {
+	pb, err := broker.NewPubSub(ctx, url, logger, broker.Prefix("alarms"))
```

Comment:
Please set JetStream Configuration using `broker.JSStreamConfig`, by default Jet Stream configuration goes to `channels`

Reply:
I have updated